queries/ruby/highlights.scm

```diff
@@ -1,5 +1,6 @@
 ; Variables
 (identifier) @variable
+(global_variable) @variable.global
```

Comment:
you would need to add that to `CONTRIBUTING.md` or adapt the CI script to only check for prefixes (here `@variable`

Reply:
Now that fallbacks are in Neovim, we can probably stick to top-level captures now?

Reply:
Thanks, I've added it.

Reply:
I would still keep some standardized prefixes, but be more liberal with some per-suffixes when it could be interesting to distinguish one feature of a language. We don't need to set highlights as `TS*` will get deprecated. I wonder whether `@variable.global` is general enough to add to more languages (probably not needed). The check coul

Reply:
@clason was your suggestion to not use captures like the proposed here?

Reply:
Sorry, I just realized the comment was ambiguous. By all means, use specialized captures in queries -- my suggestion was exactly as you said, to relax the CI check to only consider the top-level captures (so anything `@variable.foo` passes as long as `@variable` is allowed).

Reply:
@clason I just wanted to relax the capture check to only check the prefix. For testing, I added a suffix it got accepted, I wanted to introduce a typo but also the typo for a known capture got accepted. Should we still document the specializations somewhere to ensure that the capture was at least intentional? We also introduced the check because there were often synonyms or typos that were intended to be another capture. Would also be fine to allow all suffixes